sdk/state/state.go

```diff
@@ -31,8 +31,15 @@ type Channel struct {
 	observationPeriodTime      time.Duration
 	observationPeriodLedgerGap int64
 
-	startingSequence int64
-	// TODO: balances         []Amount
+	startingSequence        int64
+	iterationNumber         int64
+	iterationNumberExecuted int64
```

Comment:
Can you leave the executed field out for now and hardcode zero for the value anywhere you need it? We won't need the executed value until we support withdrawal, and I'm not sure how we will set the value yet, I think we might need to rethink it.

Reply:
commented it out for now
https://github.com/stellar/experimental-payment-channels/pull/55/commits/cf0af2d2be8645f892e362e195f235c946a2fca5